scripts/build-dev-bundle-common.sh

```diff
@@ -5,10 +5,10 @@ set -u
 
 UNAME=$(uname)
 ARCH=$(uname -m)
-NODE_VERSION=12.16.1
-MONGO_VERSION_64BIT=4.2.5
+NODE_VERSION=12.18.1
+MONGO_VERSION_64BIT=4.2.7
```

Comment:
We are using a custom build of MongoDB on Linux, so this won't work until we release a build for 4.2.7 here: https://github.com/meteor/mongodb-builder/releases.

Reply:
MongoDB 4.4 is in the release candidate stage at the moment, so maybe we should wait until 4.4 is released.

Reply:
Looks like 4.2.8 has been released recently as well. I think unless it is needless hustle I would go to the most up to date version of MongoDB 4.2.x and focus on 4.4 in the next release, which hopefully will be around the same time.

Reply:
Ok, that makes sense. However, I currently don't have access to a computer that is fast enough to build MongoDB in a reasonable amount of time. 😄 @filipenevola Does Meteor have a build farm or Linux VM that we can use to build MongoDB?

Reply:
hi @klaussner we could add a job to our Jenkins 😉 

I'm trying first locally but it is failing here 

```
error pulling image configuration: Get https://production.cloudflare.docker.com/registry-v2/docker/registry/v2/blobs/sha256/b5/b5b4d78bc90ccd15806443fb881e35b5ddba924e2f475c1071a38a3094c3081d/data?verify=1595591462-MRk%2BlaDsks9fpcIy4%2F6%2By8o2f0A%3D: Service Unavailable
```

do you know if that is a temporary problem with docker registry or if we need to update something here?

Reply:
Looks like a temporary problem. If I run the container locally, it starts compiling.